### PR TITLE
reference: update statement summary docs

### PR DESCRIPTION
**Pull request by djshow832**
### What is changed, added or deleted? 
Update documentation statement summary tables. 
I added more fields to the table and added a history table, so the documentation should be updated.

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

v3.0/v3.1/dev

### CODE REVIEW

File: dev/reference/performance/statement-summary.md [hunk 1]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`SCHEMA_NAME`：执行这类 SQL 的当前 schema。
+
+`DIGEST`：这类 SQL 的 digest。
+
+`DIGEST_TEXT`：规范化后的 SQL。
+
+`QUERY_SAMPLE_TEXT`：这类 SQL 的原 SQL 语句，多条语句只取其中一条。
+
+`TABLE_NAMES`：SQL 中涉及的所有表，多张表用“,”分隔。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`TABLE_NAMES`：SQL 中涉及的所有表，多张表用 `,` 分隔。
```

File: dev/reference/performance/statement-summary.md [hunk 2]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`INDEX_NAMES`：SQL 中使用的索引名，多个索引用“,”分隔。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`INDEX_NAMES`：SQL 中使用的索引名，多个索引用 `,` 分隔。
```

File: dev/reference/performance/statement-summary.md [hunk 3]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`SUMMARY_BEGIN_TIME`：当前统计的时间段的开始时间。
+
+`SUMMARY_END_TIME`：当前统计的时间段的结束时间。
+
+`FIRST_SEEN`：这类 SQL 的首次出现时间。
+
+`LAST_SEEN`：这类 SQL 的最后一次出现时间。
+
+在 TiDB-Server 上的执行数据：
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
在 TiDB server 上的执行数据：
```

File: dev/reference/performance/statement-summary.md [hunk 4]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`AVG_BACKOFF_TIME`：SQL 遇到需要重试的错误时在重试前的平均等待时间。
+
+`MAX_BACKOFF_TIME`：SQL 遇到需要重试的错误时在重试前的最大等待时间。
+
+`AVG_TOTAL_KEYS`：Coprocessor 扫过的 key 的平均数量。
+
+`MAX_TOTAL_KEYS`：Coprocessor 扫过的 key 的最大数量。
+
+`AVG_PROCESSED_KEYS`：Coprocessor 处理的 key 的平均数量。相比 avg_total_keys，avg_processed_keys 不包含 MVCC 的旧版本。如果 avg_total_keys 和 avg_processed_keys 相差很大，说明旧版本比较多。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`AVG_PROCESSED_KEYS`：Coprocessor 处理的 key 的平均数量。相比 `avg_total_keys`，`avg_processed_keys` 不包含 MVCC 的旧版本。如果 `avg_total_keys` 和 `avg_processed_keys` 相差很大，说明旧版本比较多。
```

File: dev/reference/performance/statement-summary.md [hunk 5]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`AVG_WRITE_KEYS`：写入 key 的平均数量。
+
+`MAX_WRITE_KEYS`：写入 key 的最大数量。
+
+`AVG_WRITE_SIZE`：写入的平均数据量，单位 byte。
+
+`MAX_WRITE_SIZE`：写入的最大数据量，单位 byte。
+
+`AVG_PREWRITE_REGIONS`：prewrite 涉及的平均 region 数量。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`AVG_PREWRITE_REGIONS`：prewrite 涉及的平均 Region 数量。
```

File: dev/reference/performance/statement-summary.md [hunk 6]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`MAX_PREWRITE_REGIONS`：prewrite 涉及的最大 region 数量。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`MAX_PREWRITE_REGIONS`：prewrite 涉及的最大 Region 数量。
```

File: dev/reference/performance/statement-summary.md [hunk 7]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`AVG_TXN_RETRY`：事务平均重试次数。
+
+`MAX_TXN_RETRY`：事务最大重试次数。
+
+`SUM_BACKOFF_TIMES`：这类 SQL 遇到需要重试的错误后的总重试次数。
+
+`BACKOFF_TYPES`：遇到需要重试的错误时的所有错误类型及每种类型重试的次数，格式为“类型:次数”，多种错误按“,”分隔。例如“txnLock:2,pdRPC:1”。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`BACKOFF_TYPES`：遇到需要重试的错误时的所有错误类型及每种类型重试的次数，格式为 `类型:次数`。如有多种错误则用 `,` 分隔，例如 `txnLock:2,pdRPC:1`。
```

File: dev/reference/performance/statement-summary.md [hunk 8]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+`AVG_AFFECTED_ROWS`：平均影响行数。
+
+`PREV_SAMPLE_TEXT`：当 SQL 是“COMMIT”时，该字段为 COMMIT 的前一条语句；否则该字段为空字符串。当 SQL 是“COMMIT”时，按 digest 和 prev_sample_text 一起分组，即不同的 prev_sample_text 的 COMMIT 也会分到不同的行。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]: “否则”是说 SQL 不是 `COMMIT` 吗？
```suggestion
`PREV_SAMPLE_TEXT`：当 SQL 是“COMMIT”时，该字段为 COMMIT 的前一条语句；否则该字段为空字符串。当 SQL 是 `COMMIT` 时，按 digest 和 `prev_sample_text` 一起分组，即不同 `prev_sample_text` 的 `COMMIT` 也会分到不同的行。
```

File: dev/reference/performance/statement-summary.md [hunk 9]
```diff
@@ -29,42 +29,191 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+## events_statements_summary_by_digest_history
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
## `events_statements_summary_by_digest_history`
```

File: dev/reference/performance/statement-summary.md [hunk 10]
```diff
@@ -94,11 +243,12 @@ SELECT avg_latency, exec_count, query_sample_text
 
 ### 哪类 SQL 的总耗时最高？
 
-如果要对系统调优，可以找出耗时最高的 3 类 SQL：
+假如上午 10:00 到 10:30 的 QPS 明显下降，可以从历史表中找出当时耗时最高的 3 类 SQL：
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
假如上午 10:00 到 10:30 的 QPS 明显下降，可以从历史表中找出当时耗时最高的三类 SQL：
```

File: dev/reference/performance/statement-summary.md [hunk 11]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
 set global tidb_enable_stmt_summary = true;
 ```
 
-`tidb_enable_stmt_summary` 有 global 和 session 两种作用域，它们的生效方式与其他系统变量不一样：
+statement summary 关闭后，系统表里的数据会被清空，下次打开后重新统计。经测试，打开后对性能几乎没有影响。
+
+还有两个控制 statement summary 的系统变量：
+
+- `tidb_stmt_summary_refresh_interval`：`events_statements_summary_by_digest` 的清空周期，单位是秒，默认值是 1800。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
- `tidb_stmt_summary_refresh_interval`：`events_statements_summary_by_digest` 的清空周期，单位是秒 (s)，默认值是 `1800`。
```

File: dev/reference/performance/statement-summary.md [hunk 12]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
+- `tidb_stmt_summary_history_size`：`events_statements_summary_by_digest_history` 保存每种 SQL 的历史的数量，默认值是 24。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
- `tidb_stmt_summary_history_size`：`events_statements_summary_by_digest_history` 保存每种 SQL 的历史的数量，默认值是 `24`。
```

File: dev/reference/performance/statement-summary.md [hunk 13]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
+
+例如，如下设置：
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
statement summary 配置示例如下：
```

File: dev/reference/performance/statement-summary.md [hunk 14]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
+
+```sql
+set global tidb_stmt_summary_refresh_interval = 1800;
+set global tidb_stmt_summary_history_size = 24;
+```
+
+那么 `events_statements_summary_by_digest` 每 30 分钟清空一次，`events_statements_summary_by_digest_history` 保存最近 12 小时的历史数据。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
以上配置生效后，`events_statements_summary_by_digest` 每 30 分钟清空一次，`events_statements_summary_by_digest_history` 保存最近 12 小时的历史数据。
```

File: dev/reference/performance/statement-summary.md [hunk 15]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
+
+这些系统变量都有 global 和 session 两种作用域，它们的生效方式与其他系统变量不一样：
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
以上两个系统变量都有 global 和 session 两种作用域，它们的生效方式与其他系统变量不一样：
```

File: dev/reference/performance/statement-summary.md [hunk 16]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
 
 - 设置 global 变量后整个集群立即生效
 - 设置 session 变量后当前 TiDB-Server 立即生效，这对于调试单个 TiDB-Server 比较有用
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
- 设置 session 变量后当前 TiDB server 立即生效，这对于调试单个 TiDB server 比较有用
```

File: dev/reference/performance/statement-summary.md [hunk 17]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
 - 优先读 session 变量，没有设置过 session 变量才会读 global 变量
 - 把 session 变量设为空字符串，将会重新读 global 变量
 
-statement summary 关闭后，系统表里的数据会被清空，下次打开后重新统计。经测试，打开后对性能几乎没有影响。
-
-由于 `events_statements_summary_by_digest` 是内存表，为了防止内存问题，需要限制保存的 SQL 条数和 SQL 的最大显示长度。这两个参数都在 config.toml 的 [stmt-summary] 类别下配置：
+由于 statement summary tables 是内存表，为了防止内存问题，需要限制保存的 SQL 条数和 SQL 的最大显示长度。这两个参数都在 config.toml 的 [stmt-summary] 类别下配置：
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
由于 statement summary tables 是内存表，为了防止内存问题，需要限制保存的 SQL 条数和 SQL 的最大显示长度。这两个参数都在 config.toml 的 `[stmt-summary]` 类别下配置：
```

File: dev/reference/performance/statement-summary.md [hunk 18]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
 
 ## 目前的限制
 
-`events_statements_summary_by_digest` 现在还存在一些限制：
+Statement summary tables 现在还存在一些限制：
 
-- 查询 `events_statements_summary_by_digest` 时，只会显示当前 TiDB-Server 的 statement summary，而不是整个集群的 statement summary。
-- statement summary 不会滚动更新。一旦 `tidb_enable_stmt_summary` 打开，SQL 信息就开始统计。随着时间的推移，statement summary 累加，所以无法查看最近一段时间内的 statement summary。所以最佳实践是，需要排查问题的时候再打开，查看一段时间内的 statement summary。
-- TiDB Server 重启后 statement summary 丢失。因为 `events_statements_summary_by_digest` 是内存表，不会持久化数据，所以一旦 Server 被重启，statement summary 随之丢失。
+- 查询 statement summary tables 时，只会显示当前 TiDB-Server 的 statement summary，而不是整个集群的 statement summary。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
- 查询 statement summary tables 时，只会显示当前 TiDB server 的 statement summary，而不是整个集群的 statement summary。
```

File: dev/reference/performance/statement-summary.md [hunk 19]
```diff
@@ -123,26 +273,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
+- TiDB Server 重启后 statement summary 丢失。因为 statement summary tables 是内存表，不会持久化数据，所以一旦 Server 被重启，statement summary 随之丢失。
```
Comment (TomShawn) [inline dev/reference/performance/statement-summary.md]:
```suggestion
- TiDB server 重启后 statement summary 会丢失。因为 statement summary tables 是内存表，不会持久化数据，所以一旦 server 被重启，statement summary 随之丢失。
```

File: dev/reference/performance/statement-summary.md [hunk 20]
```diff
@@ -123,26 +276,41 @@ statement summary 功能默认关闭，通过设置系统变量打开，例如
...
 
-这两个参数建议根据实际情况调整，不宜设置得过大。
+> **注意：**
+>
+> `tidb_stmt_summary_history_size`、`max-stmt-count`、`max-sql-length` 这些配置都影响内存占用，建议根据实际情况调整，不宜设置得过大。
 
 ## 目前的限制
 
-`events_statements_summary_by_digest` 现在还存在一些限制：
+Statement summary tables 现在还存在一些限制：
```
Comment (bb7133) [inline dev/reference/performance/statement-summary.md]:
```suggestion
目前 statement summary tables 还存在一些限制：
```
I think this avoids the upper-case dilemma.

File: dev/reference/performance/statement-summary.md [hunk 21]
```diff
@@ -1,20 +1,21 @@
...
-## events_statements_summary_by_digest 介绍
+## `events_statements_summary_by_digest`
 
-`events_statement_summary_by_digest` 是 `performance_schema` 里的一张系统表。顾名思义，它把 SQL 按 digest 分组，统计每一组的 SQL 信息。
+`events_statements_summary_by_digest` 是 `performance_schema` 里的一张系统表，它把 SQL 按 digest 分组，统计每一组的 SQL 信息。
 
 此处的 digest 与 slow log 里的 digest 一样，是把 SQL 规范化后算出的唯一标识符。
-SQL 的规范化会忽略常量、空白符、大小写的差别。也就是说，只要语法一致，就会归到同一类。
+
+SQL 的规范化会忽略常量、空白符、大小写的差别。也就是说两条 SQL 语句只要语法一致，其 digest 也会相同。
```
Comment (bb7133) [inline dev/reference/performance/statement-summary.md]: I would prefer translating the term "Normalization" to "归一化":

If you google "normalization 归一化", there are 192,000 pages, while googling "normalization 规范化" gets only 36,100 results.

File: dev/reference/performance/statement-summary.md [hunk 22]
```diff
@@ -29,42 +30,193 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+- `MAX_TXN_RETRY`：事务最大重试次数
+
+- `SUM_BACKOFF_TIMES`：这类 SQL 遇到需要重试的错误后的总重试次数
+
+- `BACKOFF_TYPES`：遇到需要重试的错误时的所有错误类型及每种类型重试的次数，格式为 `类型:次数`。如有多种错误则用 `,` 分隔，例如 `txnLock:2,pdRPC:1`
+
+- `AVG_AFFECTED_ROWS`：平均影响行数
+
+- `PREV_SAMPLE_TEXT`：当 SQL 是“COMMIT”时，该字段为 COMMIT 的前一条语句；否则该字段为空字符串。当 SQL 是 `COMMIT` 时，按 digest 和 `prev_sample_text` 一起分组，即不同 `prev_sample_text` 的 `COMMIT` 也会分到不同的行
```
Comment (bb7133) [inline dev/reference/performance/statement-summary.md]:
```suggestion
- `PREV_SAMPLE_TEXT`：当 SQL 是 “COMMIT” 时，该字段为 COMMIT 的前一条语句；否则该字段为空字符串。当 SQL 是 `COMMIT` 时，按 digest 和 `prev_sample_text` 一起分组，即不同 `prev_sample_text` 的 `COMMIT` 也会分到不同的行
```

File: dev/reference/performance/statement-summary.md [hunk 23]
```diff
@@ -29,42 +30,193 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+     SUMMARY_END_TIME: 2020-01-02 11:30:00
+            STMT_TYPE: select
+          SCHEMA_NAME: test
+               DIGEST: 0611cc2fe792f8c146cc97d39b31d9562014cf15f8d41f23a4938ca341f54182
+          DIGEST_TEXT: select * from employee where id = ?
+          TABLE_NAMES: test.employee
+          INDEX_NAMES: NULL
+          SAMPLE_USER: root@127.0.0.1
+           EXEC_COUNT: 1
+           EXEC_COUNT: 3
```
Comment (baurine) [inline dev/reference/performance/statement-summary.md]: It has 2 same `EXEC_COUNT`

File: dev/reference/performance/statement-summary.md (hunk 21 above)
Comment (lonng) [inline dev/reference/performance/statement-summary.md]:
```suggestion
SQL 的规范化会忽略常量、空白符、大小写的差别。即两条 SQL 语句只要语法一致，其 digest 也会相同。
```

File: dev/reference/performance/statement-summary.md [hunk 24]
```diff
@@ -29,42 +30,193 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
 select * from employee where id in (...) and salary between ? and ?;
 ```
 
-接下来详细看一下 `events_statements_summary_by_digest` 的表结构。
-因为 TiDB 中的很多概念不同于 MySQL，所以 `events_statements_summary_by_digest` 也与 MySQL 有一些区别。
+`events_statements_summary_by_digest` 保存的是把 SQL 的监控指标按 SQL digest 进行聚合的结果。一般来说，每一项监控指标都包含平均值和最大值。例如执行延时对应 `AVG_LATENCY` 和 `MAX_LATENCY` 两个字段，分别是平均延时和最大延时。
```
Comment (lonng) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`events_statements_summary_by_digest` 用于保存 SQL 监控指标按 digest 聚合后的结果。一般来说，每一项监控指标都包含平均值和最大值。例如执行延时对应 `AVG_LATENCY` 和 `MAX_LATENCY` 两个字段，分别是平均延时和最大延时。
```

File: dev/reference/performance/statement-summary.md [hunk 25]
```diff
@@ -29,42 +30,193 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+为了监控指标的即时性，`events_statements_summary_by_digest` 里的数据定期被清空，只展现最近一段时间内的聚合结果。清空周期由系统变量 `tidb_stmt_summary_refresh_interval` 设置。如果刚好在清空之后进行查询，显示的数据可能很少。
```
Comment (lonng) [inline dev/reference/performance/statement-summary.md]: Is there any variable to indicate the refresh time? If any, I think we should document it too, or else it's better to add one.

File: dev/reference/performance/statement-summary.md [hunk 26]
```diff
@@ -29,42 +30,193 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+因为 TiDB 中的很多概念不同于 MySQL，所以 TiDB 中 `events_statements_summary_by_digest` 的表结构与 MySQL 中的有很大区别。
 
-查询 `events_statements_summary_by_digest` 的输出示例：
+`events_statements_summary_by_digest` 表中有 64 个字段，以下为部分查询结果：
```
Comment (lonng) [inline dev/reference/performance/statement-summary.md]: It's better to remove the `64` for maintaining purpose.

File: dev/reference/performance/statement-summary.md (hunk 22 above)
Comment (lonng) [inline dev/reference/performance/statement-summary.md]:
```suggestion
- `PREV_SAMPLE_TEXT`：当 SQL 是 `COMMIT` 时，该字段为 COMMIT 的前一条语句；否则该字段为空字符串。当 SQL 是 `COMMIT` 时，按 digest 和 `prev_sample_text` 一起分组，即不同 `prev_sample_text` 的 `COMMIT` 也会分到不同的行
```

File: dev/reference/performance/statement-summary.md [hunk 27]
```diff
@@ -29,42 +30,193 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
...
+
+- `BACKOFF_TYPES`：遇到需要重试的错误时的所有错误类型及每种类型重试的次数，格式为 `类型:次数`。如有多种错误则用 `,` 分隔，例如 `txnLock:2,pdRPC:1`
+
+- `AVG_AFFECTED_ROWS`：平均影响行数
+
+- `PREV_SAMPLE_TEXT`：当 SQL 是“COMMIT”时，该字段为 COMMIT 的前一条语句；否则该字段为空字符串。当 SQL 是 `COMMIT` 时，按 digest 和 `prev_sample_text` 一起分组，即不同 `prev_sample_text` 的 `COMMIT` 也会分到不同的行
+
+## `events_statements_summary_by_digest_history`
+
+`events_statements_summary_by_digest_history` 的表结构与 `events_statements_summary_by_digest` 完全相同，不同的是前者保存了历史时间段的数据。通过历史数据，可以排查过去出现的异常，也可以对比不同时间的监控指标。
```
Comment (lonng) [inline dev/reference/performance/statement-summary.md]:
```suggestion
`events_statements_summary_by_digest_history` 的表结构与 `events_statements_summary_by_digest` 完全相同，用于保存历史时间段的数据。通过历史数据，可以排查过去出现的异常，也可以对比不同时间的监控指标。
```

File: dev/reference/performance/statement-summary.md (hunk 25 above)
Comment (djshow832) [reply dev/reference/performance/statement-summary.md]: It's already documented, under the '配置' title below.

Comment (lonng) [reply dev/reference/performance/statement-summary.md]: OK

**Comment by djshow832** [comment]
As plan digest has been merged to the release branches, I added description about plan digest to this PR, the commit is https://github.com/pingcap/docs-cn/pull/2173/commits/0db4d94f5f6b9ed767fa61de4e7f5148a19271cf
PTAL @TomShawn @bb7133 @lonng

**Comment by djshow832** [comment]
Please approve it. @TomShawn

File: dev/reference/performance/statement-summary.md [hunk 28]
```diff
@@ -23,48 +22,145 @@ SELECT * FROM employee WHERE id IN (1, 2, 3) AND salary BETWEEN 1000 AND 2000;
...
+
+- `EXEC_COUNT`：这类 SQL 的总执行次数
+- `SUM_LATENCY`：这类 SQL 的总延时
+- `MAX_LATENCY`：这类 SQL 的最大延时
+- `MIN_LATENCY`：这类 SQL 的最小延时
+- `AVG_LATENCY`：这类 SQL 的平均延时
+- `AVG_PARSE_LATENCY`：解析器的平均延时
+- `MAX_PARSE_LATENCY`：解析器的最大延时
+- `AVG_COMPILE_LATENCY`：优化器的平均延时
+- `MAX_COMPILE_LATENCY`：优化器的最大延时
```
Comment (baurine) [inline dev/reference/performance/statement-summary.md]: sorry just remember to ask, what's the unit for these time metrics, I guess it should be "ms" or "us", right? can we put the units for all metrics?

**Pull request merged**